app/models/trainee.rb

```diff
@@ -326,6 +326,12 @@ def course_duration_in_years
     ((itt_end_date - itt_start_date) / 365).ceil
   end
 
+  def course_programme_year
```

Comment:
Knowing that DTTP integration will end soon, can we move this method into `Dttp::Params::PlacementAssignment` so that when it comes to stripping out all the DTTP code, we don't have to touch the `Trainee` model. I think `course_duration_in_years()` could also move over.